.editorconfig

```diff
@@ -7,11 +7,11 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
-indent_style = space
-indent_size = 2
+indent_style = tab
+indent_size = 4
```

Comment:
I always thought that we preferred tabs instead of spaces in resin. I'll set the size back to 2 but can we keep the tabs ?